### ISSUE TITLE
When moving mouse with drawing tablet in CURSOR_DISABLED mode, mouse coordinates rapidly change

### ISSUE THREAD

**Issue by LapisSea** [glfw/glfw#1265]
Hi!
When I was implementing mouse scroll, I wanted to try out if the mouse scroll delta would work when I would use my wacom drawing tablet. 
Before getting any readings I was greeted with insane mouse movement any time I would invoke mouse move with the tablet.
(here is the log of the mouse position)
![image](https://user-images.githubusercontent.com/8900407/39828972-3df4e6f0-53bd-11e8-8c42-0d98837db5bc.png)

I tried to rearrange and change my code to fix this, but no luck. My speculation is that GLFW is not properly handling input devices with sub pixel accuracy.

This does not occur in hidden or normal mode.

OS: windows 10
Displays: dual screen 1080p monitors
GLFW version: 3.3.0 Win32 WGL EGL VisualC DLL
Occurs on device:  wacom intuos p s

Note: I am using LWJGL 3.1.6 bindings and Vulkan context

Short demonstration of the issue: 
https://www.youtube.com/watch?v=Qw7Bowb6eOE

**Comment by elmindreda** [comment]
Cannot reproduce on Windows 7. Will try on Windows 10 asap.

**Comment by orange451** [comment]
Do you have a compiled version of your application? I also have a wacom tablet somewhere, I could give it a test.

**Comment by LapisSea** [comment]
Yes here. https://drive.google.com/open?id=1AQAWqMdcTRy_2X7Em7qUlzhUXkpJVH-7
Make sure you have Java 10 (don't have a detection of jvm and embedded jre fallback implemented yet)

Extract the folder. There will be 3 bats, each representing what cursor mode to use when window is focused.

**Comment by u56237z** [comment]
Experiencing this exact same problem, using 3.3.2 both on Windows and Linux, with a Wacom Intuos.

any workaround?
